gen/briefinggen.py

```diff
@@ -79,20 +211,13 @@ def add_tanker(self, tanker: TankerInfo) -> None:
         """
         self.tankers.append(tanker)
 
-    def generate(self) -> None:
-        """Generates the mission information."""
-        raise NotImplementedError
-
+    def add_frontline(self, frontline: FrontLineInfo) -> None:
+        '''Adds a frontline to the briefing
 
-class BriefingGenerator(MissionInfoGenerator):
-
-    def __init__(self, mission: Mission, conflict: Conflict, game):
-        super().__init__(mission)
-        self.conflict = conflict
-        self.game = game
-        self.title = ""
-        self.description = ""
-        self.dynamic_runways: List[RunwayData] = []
+        Arguments:
+            frontline {FrontLineInfo} -- [description]
```

Comment:
I don't think you finished writing this :) Either finish it or remove the `Args` section (we haven't been very diligent at writing them, so either way works for me)

Type info is usually omitted in the argument list since the function declaration already has that covered.

Reply:
fixed